src/a-d/Alma/Alma.BufferList.cs

```diff
@@ -3,7 +3,7 @@ namespace Skender.Stock.Indicators;
 /// <summary>
 /// Arnaud Legoux Moving Average (ALMA) from incremental reusable values.
 /// </summary>
-public class AlmaList : List<AlmaResult>, IAlma, IBufferList, IBufferReusable
+public class AlmaList : BufferList<AlmaResult>, IAlma, IBufferReusable
```

Comment:
For all of the implementations, I'd like this to be in increasing refinement order for interfaces, so `IAlma, IBufferReusable` should be `IBufferReusable, IAlma`.  Update everywhere and in the instructions if it'd depicted the other way around.